examples/default/domain/aggregate.go

```diff
@@ -75,7 +75,7 @@ func (i *InvitationAggregate) HandleCommand(command eh.Command) error {
 			return nil
 		}
 
-		i.StoreEvent(&InviteAccepted{i.AggregateID()})
+		i.StoreEvent(&InviteAccepted{i.AggregateID().(eh.UUID)})
```

Comment:
If you want to see, I create a new branch to test this changes
https://github.com/rodrigocorsi2/eventhorizon/tree/uuid_test